imagebuildah/chroot_symlink.go

```diff
@@ -154,6 +158,9 @@ func modTimeIsGreater(rootdir, path string, historyTime string) (bool, error) {
 		if info.Mode()&os.ModeSymlink == os.ModeSymlink {
 			// Evaluate any symlink that occurs to get updated modified information
 			resolvedPath, err := filepath.EvalSymlinks(path)
+			if err != nil && os.IsNotExist(err) {
+				return errDanglingSymlink
```

Comment:
If `errDanglingSymlink` becomes a standard-library-produced `error`, this will probably want to be wrapped with `errors.Wrapf()`...

Reply:
Fixed.